src/pybind/mgr/volumes/fs/stats_util.py

```diff
@@ -190,7 +191,7 @@ def _get_info_for_all_clones(self):
                                  f'Printing the exception: {e}')
                         continue
                     except VolumeException as e:
-                        if e.error_str != 'error fetching subvolume metadata':
+                        if e.errno != -errno.EINVAL:
```

Comment:
Looks fine. Is this the only errno to handled?

Reply:
I don't get you. You mean only errno when `'error fetching subvolume metadata` is raised?

Reply:
Yeah. Does this _only_ needs to handle EINVAL?

Reply:
Yep. No other errno can occur -

```
$ grep -rnI 'error fetching subvolume metadata' src/pybind/mgr/volumes/fs/
src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py:59:            raise VolumeException(-errno.EINVAL, "error fetching subvolume metadata")
src/pybind/mgr/volumes/fs/operations/versions/subvolume_v1.py:674:            raise VolumeException(-errno.EINVAL, "error fetching subvolume metadata")
src/pybind/mgr/volumes/fs/stats_util.py:193:                        if e.error_str != 'error fetching subvolume metadata':

```